tests/foreman/sys/test_katello_certs_check.py

```diff
@@ -21,6 +21,54 @@
 
 import pytest
 
+import robottelo.constants as constants
+from robottelo.config import settings
+
+
+@pytest.mark.e2e
+def test_positive_install_sat_with_katello_certs(certs_data, sat_ready_rhel):
+    """Install Satellite with custom certs.
+
+    :id: 47e3a57f-d7a2-40d2-bbc7-d1bb3d79a7e1
+
+    :steps:
+
+        1. Generate the custom certs on RHEL machine
+        2. Install satellite with custom certs
+        3. Assert output does not report SSL certificate error
+        4. Assert all services are running
+
+
+    :expectedresults: Satellite should be installed using the custom certs.
+
+    :CaseAutomation: Automated
+    """
+    version = sat_ready_rhel.os_version.major
+    sat_ready_rhel.download_repofile(product='satellite', release=settings.server.version.release)
+    sat_ready_rhel.register_to_cdn()
+    for repo in getattr(constants, f"OHSNAP_RHEL{version}_REPOS"):
+        sat_ready_rhel.enable_repo(repo, force=True)
+    sat_ready_rhel.execute('yum -y update')
+    result = sat_ready_rhel.execute(
+        'dnf -y module enable satellite:el8 && dnf -y install satellite'
+    )
+    assert result.status == 0
+    command = (
+        'satellite-installer --scenario satellite '
+        f'--certs-server-cert "/root/{certs_data["cert_file_name"]}" '
+        f'--certs-server-key "/root/{certs_data["key_file_name"]}" '
+        f'--certs-server-ca-cert "/root/{certs_data["ca_bundle_file_name"]}" '
```

Comment:
Can we use InstallerCommand here and sat_ready_rhel.install(..)?

Reply:
Updated to use `InstallerCommand` 
but not `install`
```
        return self.execute(installer_obj.get_command(), timeout=0)
```
timeout=0     vs.    timeout=220000

Reply:
Further updates.
Now using `install(InstallerCommand())` combo :-)

Reply:
Reverting back to using just `InstallerCommand()`
By using `install()` where timeout is set to `0` the command has timed out after 10min

0 doesn't mean unlimited as is wrongly stated on many places in the code

Reply:
@JacobCallahan ^ fyi

Reply:
Then it seems that
```
By default or if timeout set is zero, blocking calls do not time out. :param timeout: Milliseconds to wait before timeout.
```
statement from ssh2-python docs does not hold true.
https://ssh2-python.readthedocs.io/en/latest/session.html?highlight=Session#ssh2.session.Session.set_timeout